### PR TITLE
taking one or more files for regions for assigns

### PR DESCRIPTION
**Pull request by clarax**
This is for https://issues.apache.org/jira/browse/HBASE-23927

HBCK2.java changes are mostly format cleanup except marked area.

### CODE REVIEW

File: hbase-hbck2/src/main/java/org/apache/hbase/HBCK2.java [hunk 1]
```diff
@@ -17,6 +17,9 @@
  */
 package org.apache.hbase;
 
+import java.io.BufferedReader;
+import java.io.File;
```
Comment (clarax) [inline hbase-hbck2/src/main/java/org/apache/hbase/HBCK2.java:21]: actual change

File: hbase-hbck2/src/main/java/org/apache/hbase/HBCK2.java [hunk 2]
```diff
@@ -269,21 +274,23 @@ int setRegionState(ClusterConnection connection, String region,
...
-        formatNameSpaceTableParam(nameSpaceOrTable));
+              formatNameSpaceTableParam(nameSpaceOrTable));
     } catch (IOException e) {
       LOG.error("Error adding missing regions: ", e);
       throw e;
     }
   }
 
-  List<Long> assigns(Hbck hbck, String [] args) throws IOException {
+  List<Long> assigns(Hbck hbck, String[] args) throws IOException {
```
Comment (clarax) [inline hbase-hbck2/src/main/java/org/apache/hbase/HBCK2.java:288]: Actual changes.

File: hbase-hbck2/src/main/java/org/apache/hbase/HBCK2.java [hunk 3]
```diff
@@ -434,23 +463,27 @@ private static void usageAddFsRegionsMissingInMeta(PrintWriter writer) {
...
     writer.println("   namespace 'n2':");
     writer.println("     $ HBCK2 " + ADD_MISSING_REGIONS_IN_META_FOR_TABLES +
-        " default:tbl_1 n1:tbl_2 n2");
+            " default:tbl_1 n1:tbl_2 n2");
     writer.println("   Returns HBCK2  an 'assigns' command with all re-inserted regions.");
     writer.println("   SEE ALSO: " + REPORT_MISSING_REGIONS_IN_META);
     writer.println("   SEE ALSO: " + FIX_META);
   }
 
   private static void usageAssigns(PrintWriter writer) {
```
Comment (clarax) [inline hbase-hbck2/src/main/java/org/apache/hbase/HBCK2.java:472]: actual changes

**Comment by busbey** [comment]
Please pull the cleanup into its own PR

**Comment by clarax** [comment]
> Please pull the cleanup into its own PR

Thank you. Closed and opened a new PR https://github.com/apache/hbase-operator-tools/pull/64

**Pull request closed without merging**